sdk/instruction/src/lib.rs

```diff
@@ -296,3 +296,23 @@ pub struct ProcessedSiblingInstruction {
     /// Number of AccountMeta structures
     pub accounts_len: u64,
 }
+
+/// Borrowed version of `AccountMeta`.
+///
+/// This struct is used by the runtime when constructing the sysvar. It is not
+/// useful to Solana programs.
```

Comment:
nit: to be clear about where it's used
```suggestion
/// This struct is used by the runtime when constructing the instructions sysvar. It is not
/// useful to Solana programs.
```